docroot/modules/custom/civictheme_content/modules/civictheme_content_corporate/civictheme_content_corporate.install

```diff
@@ -0,0 +1,15 @@
+<?php
+
+/**
+ * @file
+ * Install for civictheme_content_corporate module.
+ */
+
+/**
+ * Implements hook_install().
+ */
+function civictheme_content_corporate_install() {
+  civictheme_content_clear_menu('civictheme-primary-navigation');
```

Comment:
the way this works now:
1. Theme is installed
2. Provisioner runs from CLI to create content, including menu links
3. `civictheme_content` module is installed to enable demo content - we need to clear out anything created in the step 2 so menus are not polluted with links created during a manual provisioning and as a part of the default content.